src/schema/extension.schema.json

```diff
@@ -16,6 +16,11 @@
       "type": "string",
       "maxLength": 280
     },
+    "icon": {
+      "description": "The extension icon in data URI format (must begin with data:image/png).",
+      "type": "string",
+      "format": "^data:image/png"
```

Comment:
Shouldn't there be a separator at the end? I am wondering if I could trick this regex with something like `image/png+svg`, `image/png*` or `image/png,svg`

Reply:
I started with `image/svg+xml`:

```html
<img src="data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PScwIDAgMTA1IDkzJyB4bWxucz0naHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmcnPgo8cGF0aCBkPSdNNjYsMGgzOXY5M3pNMzgsMGgtMzh2OTN6TTUyLDM1bDI1LDU4aC0xNmwtOC0xOGgtMTh6JyBmaWxsPScjRUQxQzI0Jy8+Cjwvc3ZnPgo=">
```

Then I tried your examples, but wasn't able to circumvent the regex. Check out https://jsfiddle.net/4m86L0aj/4/

Reply:
I believe it, but why not just add the `;` so it's definitely PNG? Takes less time than to create a JSFiddle, and handles cases I didn't think of :)

Reply:
Hmm, the [spec](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs) says the syntax is `data:[<mediatype>][;base64],<data>`, so `;base64` is optional. I tightened the regex to `data:image\/png(;base64)?,`.

Reply:
Ah, got it